src/components/skiplink-list/skiplink-list.ts

```diff
@@ -70,21 +62,10 @@ export class SbbSkiplinkListElement extends SlotChildObserver(LitElement) {
     this._links = links;
   }
 
-  public override connectedCallback(): void {
-    super.connectedCallback();
-    this._handlerRepository.connect();
-  }
-
-  public override disconnectedCallback(): void {
-    super.disconnectedCallback();
-    this._handlerRepository.disconnect();
-  }
-
   protected override render(): TemplateResult {
     let ariaLabelledByAttribute: Record<string, string> = {};
-    if (this._namedSlots.title || this.titleContent) {
-      ariaLabelledByAttribute = { 'aria-labelledby': 'sbb-skiplink-list-title-id' };
-    }
+    ariaLabelledByAttribute = { 'aria-labelledby': 'sbb-skiplink-list-title-id' };
```

Comment:
I'd vote for inline this in the template

Reply:
Totally agreeable